app/game/memoryGame/script.js

```diff
@@ -0,0 +1,83 @@
+const canvas0 = document.getElementById("canvas0"),
+canvas1 = document.getElementById("canvas1"),
+canvas2 = document.getElementById("canvas2"),
+canvas3 = document.getElementById("canvas3"),
+canvas4 = document.getElementById("canvas4"),
+canvas5 = document.getElementById("canvas5"),
+canvas6 = document.getElementById("canvas6"),
+canvas7 = document.getElementById("canvas7"),
+canvas8 = document.getElementById("canvas8");
```

Comment:
Would there be anything else with id "canvasX"? I think a more descriptive Id could help